### DISCUSSION TITLE
text-decoration-color

### DISCUSSION THREAD

**Discussion by DanielRiveraHQ** [tailwindlabs/tailwindcss#3283] [Ideas]
I would love to have text-decoration-color option to be different than the text. 

Thank you for your hard work!

**Comment by gabrielloeb** [comment]
Daniel, I hope you don’t mind me piggybacking on your suggestion.

I’d love to see `text-decoration-color` added to `.transition` and `.transition-colors` as well.

I imagine this would be pretty straightforward and I’d be glad to contribute.

https://caniuse.com/?search=text-decoration-color

**Comment by marcia-lima** [comment]
The following discussions might be related to yours:

- [#2166](https://github.com/tailwindlabs/tailwindcss/discussions/2166)
- [#2665](https://github.com/tailwindlabs/tailwindcss/discussions/2665)
